pytorch_blade/mnist.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2022 The BladeDISC Authors. All rights reserved.
```

Comment:
It seems just copy `torch_disc` files to `pytorch_blade` folder? Can you add more explanation?